src/__configuration__/themes/schedule.ts

```diff
@@ -1,87 +1,15 @@
-import { ThemeOptions } from '@material-ui/core';
-import { HalloweenTheme } from './Halloween';
-import * as PXBThemes from '@pxblue/react-themes';
-
-// landing page banner background images
-import circles from '../../app/assets/themes/circles.svg';
-import castles from '../../app/assets/themes/castles.svg';
-import ghosts from '../../app/assets/themes/halloween-appbar-tile.svg';
-import { CSSProperties } from '@material-ui/core/styles/withStyles';
-
-type SiteConfig = {
-    /**
-     * A MUI theme to be applied during the scheduled period
-     */
-    theme: ThemeOptions;
-
-    /**
-     * Configuration around the banner image in the landing page
-     */
-    landingPageBanner: {
-        src: string;
-    } & CSSProperties;
-
-    /**
-     * A theme-specific class name to be used by SASS
-     */
-    className?: string;
-
-    /**
-     * Configuration around the app bar background image
-     */
-    appBarBackground?: {
-        src: string;
-    } & CSSProperties;
-};
-
-type Schedule = {
-    /**
-     * Start date of the duration (inclusive)
-     */
-    start: Date;
-
-    /**
-     * Ending date of the duration (inclusive)
-     */
-    end: Date;
-
-    /**
-     * If start & end is fulfilled, what configuration to apply to the site
-     */
-    config: SiteConfig;
-};
-
-export const schedule: Schedule[] = [
-    {
-        start: new Date(0, 9, 14), // Oct 14
-        end: new Date(0, 10, 13), // Nov 13
-        config: {
-            theme: HalloweenTheme,
-            landingPageBanner: {
-                src: castles,
-                backgroundSize: 'cover',
-                backgroundPosition: 'bottom',
-                backgroundRepeat: 'no-repeat',
-            },
-            className: 'halloween',
-            appBarBackground: {
-                src: ghosts,
-            },
-        },
-    },
-    /*
-    Example of a new year theme (wrapped to the next year)
-    {
-        start: new Date(0, 11, 31), // Dec 31
-        end: new Date(1, 0, 3),     // Jan 03
-        config: {
-            theme: NewYearsDayTheme,
-            landingPageBanner: Fireworks,
-        },
-    },
-    */
-];
-
+import { HalloweenSchedule } from './halloween';
+import { MayTheForthSchedule } from './may-the-forth';
+import { ChristmasSchedule } from './christmas';
+import { defaultTheme } from './default';
+import { Schedule, SiteConfig } from './types';
+
+// Add more holiday themes here
+export const schedule: Schedule[] = [ChristmasSchedule, HalloweenSchedule, MayTheForthSchedule];
+
+/**
+ * @returns a site configuration given the currently applied holiday theme
+ */
```

Comment:
Might be nice to see a comment somewhere that shows the schedule in plain english, e.g.:
```
/**
* ...
* Dec 23 - Dec 26: Christmas
* Dec 29 - Jan 2: New Years
**/
```